src/main/java/pl/ergohestia/ehj1/ivesta/services/VehicleValidator.java

```diff
@@ -30,32 +30,32 @@ public boolean isVehicleValid() {
     }
 
     public void validateSeats() {
-        if (!(vehicle.getNumberOfSeats() > 0)) {
+        if (!(vehicleDto.getNumberOfSeats() > 0)) {
```

Comment:
Zakręcona logika. Unikaj negacji. <=0 oznacza to samo. ale na czuja wygląda nieintuicyjnie. Jeśli ujemna liczba miejsc, to dodaj? A może `addParameterToList` trudno się domyślić co robi.